src/Mutator/Boolean/TrueValue.php

```diff
@@ -54,12 +60,29 @@ public function mutate(Node $node)
         return new Node\Expr\ConstFetch(new Node\Name('false'));
     }
 
-    protected function mutatesNode(Node $node): bool
+    protected function mutatesNode(Node $node, array $mutatorSettings): bool
     {
         if (!($node instanceof Node\Expr\ConstFetch)) {
             return false;
         }
 
-        return $node->name->toLowerString() === 'true';
+        if ($node->name->toLowerString() !== 'true') {
+            return false;
+        }
+
+        $parentNode = $node->getAttribute(ParentConnectorVisitor::PARENT_KEY);
+        $grandParentNode = $parentNode !== null ? $parentNode->getAttribute(ParentConnectorVisitor::PARENT_KEY) : null;
+
+        if ($grandParentNode instanceof Node\Expr\FuncCall) {
```

Comment:
We can reduce indentation by inverting this if, and returing  true much earlier e.g.

```php
if (! $grandParentNode instanceof Node\Expr\FuncCall) {
    return true;
}
// Rest of the code ...
```

Reply:
done